Gemfile

```diff
@@ -32,6 +32,7 @@ gem "with_advisory_lock", "~> 4"
 group :development do
   gem "brakeman", "~> 4"
   gem "listen", "~> 3"
+  gem "pry", "~> 0.12"
```

Comment:
We've already got [byebug](https://github.com/deivid-rodriguez/byebug) for this a few lines down in the dev/test section.

Reply:
Ahh nice! Any reason why we're using byebug instead of pry? I haven't come across it before.

(Will be removing this commit though - pointless having two debug gems!).